README.md

```diff
@@ -176,6 +176,18 @@ If you want to override all the values with the new one you can configure `multi
 
 An example of this is `INFRA-CONTAINER` where the tag `container.state` will always display the last state (`running`, `stopped`, etc) instead of a list of all the states the entity has gone through.
 
+You can also configure the key-name that you want for the tags we synthesize from your telemetry attributes. 
```

Comment:
`s/tags/entity tags` (just to make it extra explicit as the fact that we have two different sets of tags/attributes can be confusing for people)

Reply:
Names sense, will change, thanks!